src/Diagnostics.ModelsAndUtils/Models/Storage/DiagConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Diagnostics.ModelsAndUtils.Models.Storage
+{
+    public class DiagConfiguration: TableEntity
```

Comment:
Should we rename this something more accurate to the data its holding? When I see DiagConfiguration I'm thinking our platform configuration.

Reply:
How about `DetectorConfiguration` ?

Reply:
Are we planning to add different types of configurations into `DetectorConfiguration` otherwise maybe `KustoMappingConfiguration` is best?

Reply:
I thought of keeping this generic and flexible to add anything in future if needed. That's why the partitionkey is set to "KustoClusterMapping". We can always add anything in future by setting value for partitionkey

Reply:
Awesome

Reply:
Maybe not call it `DetectorConfiguration`. Just thinking about what kind of data this table holds, KustoMapping data can change anytime during the runtime and I assume any other type of data coming into this table in the future should also by some form of dynamic data. Anything that is static will just be configuration data in appsettings and keyvault. So maybe call it `DetectorRuntimeConfiguration` or any name you can think of that is configuration focused and dynamic.